packages/storybook/src/utils/versions.ts

```diff
@@ -1,4 +1,5 @@
 export const storybookAngularVersion = '5.1.11';
 export const storybookAddonKnobsVersion = '5.1.11';
+export const storybookAddonKnobsTypesVersion = '5.0.3';
```

Comment:
Why are default addons required? It's not needed to set it up manually so shouldn't be required to do it programmatically using schematics

Reply:
I'm using the knobs addon in the auto-generated stories.  That's probably the first addon most people will include.

Reply:
Realistically, are there any usages that don't use knobs? I don't think it's a bad default.